src/Compiler/CompilerUtils.cpp

```diff
@@ -309,26 +325,37 @@ void setTargetArch(const std::string &arch) { march = arch; }
 void setTargetTriple(const std::string &triple) { mtriple = triple; }
 void setOptLevel(const OptLevel level) { OptimizationLevel = level; }
 OptLevel getOptLevel() { return OptimizationLevel; }
```

Comment:
OptLevel is the only one that has just a plain get. Is it needed?

Reply:
Yeah, we should avoid the "asymmetry".  Best to leave these untouched in this PR though because  @AlexandreEichenberger is restructuring the option processing via PR https://github.com/onnx/onnx-mlir/pull/1167